vnext/Microsoft.ReactNative.Managed.CodeGen/App.cs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
> All rights reserved. [](start = 39, length = 21)

Please remove the ' All rights reserved.' from all new files. It is changed by PR #5083

Reply:
I'll wait for Andrew to merge that change and fixup after that to ensure I got all via a full text search.